receiver/mongodbatlasreceiver/README.md

```diff
@@ -15,6 +15,8 @@ below both values are being pulled from the environment.
 - `public_key`
 - `private_key`
 - `granularity` (default `PT1M` - See [MongoDB Atlas Documentation](https://docs.atlas.mongodb.com/reference/api/process-measurements/))
+- `retry_interval` (in milliseconds, default 750ms)
+- `retry_attempts` (interval doubles with each attempt)
```

Comment:
It would be better for UX if we can have a section that has a similar setup to the `retry_on_failure` section from the `exporterhelper`: https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/exporterhelper#configuration. Since we use the same library here, it should be easy to do